examples/sdl_opengl/main.odin

```diff
@@ -24,38 +24,38 @@ main :: proc() {
     context.logger = log.create_console_logger(opt = logger_opts);
 
     log.info("Starting SDL Example...");
-    init_err := sdl.init(.Video);
-    //defer sdl.quit();
+    init_err := sdl.Init(sdl.INIT_VIDEO);
```

Comment:
I'm pretty new to odin, for some reason I couldn't figure out how to make `sdl.` implicit here.